nais-prod.yaml

```diff
@@ -79,8 +79,12 @@ spec:
       - type: POSTGRES_12 # IF This is changed, all data will be lost. Read on nais.io how to upgrade
         diskSize: 50
         diskAutoresize: true
-        tier: db-custom-2-7680
+        tier: db-g1-small
```

Comment:
Dette blir ikke for "lite" ?

Reply:
Vi satte litt høy minne ifbm migrering av db til gcp. Tror mottak skal klare seg med mindre, men vi må ha db med høy availability, og får vi det default med db-g1-small?

Reply:
Ser ut som at man får allokert flere concurrent connections i alle fall, sammenlignet med micro

Reply:
Etter samtale med Ole så endret jeg til `custom-1`
Vi bruker connection pool med 10-20 connections, så skal nog gå fint :)